src/styles/sidebar/components/share-annotations-panel.scss

```diff
@@ -0,0 +1,58 @@
+.share-annotations-panel {
+  color: $grey-5;
+
+  .share-annotations-panel__intro {
+    color: $grey-7;
+    font-weight: 500;
+  }
+  .share-annotations-panel__input {
+    display: flex;
+  }
+  .share-annotations-panel__form-input {
+    border-radius: 0;
+  }
+  .share-annotations-panel__copy-btn {
+    position: relative;
```

Comment:
May not need position or display here?

Reply:
Indeed not, thanks. Was moving fast here.